src/main.cpp

```diff
@@ -10,34 +10,28 @@
 
 #include <iostream>
 #include <string>
+#include <map>
 #include <boost/asio.hpp>
 #include <boost/bind.hpp>
 #include "server.hpp"
 #include "nginx-configparser/config_parser.h"
+#include "server_options.hpp"
 
-// Expected format of config file:
-// server {
-//   listen 8080;
-//   path /echo EchoHandler;
-//   path /static StaticFileHandler {
-//      root static;
-//   }
-// }
-
-struct server_options {
-  std::string port;
-  std::string static_file_root;
-  std::string echo_handler;
-  std::string static_handler;
-};
+namespace http{
```

Comment:
Not sure, but I think we can encompass the whole code in `namespace http` and `namespace server` like we do in the other files to avoid having to write things like `http::server::server_options` all over the place later (I could be wrong though, might be different since this isn't a class)